pages/index.js

```diff
@@ -54,5 +54,9 @@ export async function getServerSideProps() {
       articles: articles?.data?.node?.childContentItemsConnection?.edges,
       sermon: sermonRequest?.data?.node,
     },
+    // Next.js will attempt to re-generate the page:
+    // - When a request comes in
+    // - At most once every second
```

Comment:
so do it attempt to regen when a new request comes in OR every 60 sec?

Reply:
it attempts to regen if a request comes in AND it has been more than 60 sec since the last request

Reply:
we could make this value way higher - it depends on often we want these queries to be rerun

Reply:
nice